lib/matplotlib/tests/test_axes.py

```diff
@@ -2905,9 +2931,6 @@ def test_scatter_different_shapes(self, fig_test, fig_ref):
 
     @pytest.mark.parametrize('c_case, re_key', params_test_scatter_c)
     def test_scatter_c(self, c_case, re_key):
-        def get_next_color():
-            return 'blue'  # currently unused
-
```

Comment:
Let's not move the `get_next_color()` functions out of the tests. This produces an indirection (harder to read) and implicit coupling of the tests. There's also value in the "currently unused" comment in here.

Compared to this, the repetition of two lines is clearly the lesser evil.

Reply:
Rolled back.